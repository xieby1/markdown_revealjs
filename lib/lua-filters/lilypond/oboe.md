---
lilypond:
  relativize: yes
---

# The Oboe
This instrument has a range of two octaves and fifth and is written in the
G-clef:

```{.lilypond .ly-fragment ly-caption="The range of the oboe" ly-name="oboe"}
\relative c' {
    \override Staff.TimeSignature #'stencil = ##f
    \cadenzaOn
    b4 c4 d4 e4 f4 g4 a4 b4 c4 d4 e4 f4 g4 a4 b4 c4 d4 e4 f4 \bar "|."
    \cadenzaOff
}
```

The two highest notes should be used with caution; the F in particular is risky
when it enters abruptly. Some oboes have the low Bâ™­ `\new Staff { \override
Staff.TimeSignature #'stencil = ##f \cadenzaOn bes4 \cadenzaOff }`{.lilypond
ly-caption="A low note" ly-name="b-flat"}; but this tone, not generally
available on this instrument, should better be avoided.
