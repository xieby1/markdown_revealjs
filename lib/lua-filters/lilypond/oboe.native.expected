[ Header
    1
    ( "the-oboe" , [] , [] )
    [ Str "The" , Space , Str "Oboe" ]
, Para
    [ Str "This"
    , Space
    , Str "instrument"
    , Space
    , Str "has"
    , Space
    , Str "a"
    , Space
    , Str "range"
    , Space
    , Str "of"
    , Space
    , Str "two"
    , Space
    , Str "octaves"
    , Space
    , Str "and"
    , Space
    , Str "fifth"
    , Space
    , Str "and"
    , Space
    , Str "is"
    , Space
    , Str "written"
    , Space
    , Str "in"
    , Space
    , Str "the"
    , SoftBreak
    , Str "G-clef:"
    ]
, Para
    [ Image
        ( "" , [ "lilypond-image-standalone" ] , [] )
        [ Str "The range of the oboe" ]
        ( "./oboe.png"
        , "fig:\\relative c' { \\override Staff.TimeSignature #'stencil = ##f \\cadenzaOn b4 c4 d4 e4 f4 g4 a4 b4 c4 d4 e4 f4 g4 a4 b4 c4 d4 e4 f4 \\bar \"|.\" \\cadenzaOff }"
        )
    ]
, Para
    [ Str "The"
    , Space
    , Str "two"
    , Space
    , Str "highest"
    , Space
    , Str "notes"
    , Space
    , Str "should"
    , Space
    , Str "be"
    , Space
    , Str "used"
    , Space
    , Str "with"
    , Space
    , Str "caution;"
    , Space
    , Str "the"
    , Space
    , Str "F"
    , Space
    , Str "in"
    , Space
    , Str "particular"
    , Space
    , Str "is"
    , Space
    , Str "risky"
    , SoftBreak
    , Str "when"
    , Space
    , Str "it"
    , Space
    , Str "enters"
    , Space
    , Str "abruptly."
    , Space
    , Str "Some"
    , Space
    , Str "oboes"
    , Space
    , Str "have"
    , Space
    , Str "the"
    , Space
    , Str "low"
    , Space
    , Str "B\9837"
    , Space
    , Image
        ( "" , [ "lilypond-image-inline" ] , [] )
        [ Str "A low note" ]
        ( "./b-flat.png"
        , "\\new Staff { \\override Staff.TimeSignature #'stencil = ##f \\cadenzaOn bes4 \\cadenzaOff }"
        )
    , Str ";"
    , Space
    , Str "but"
    , Space
    , Str "this"
    , Space
    , Str "tone,"
    , Space
    , Str "not"
    , Space
    , Str "generally"
    , SoftBreak
    , Str "available"
    , Space
    , Str "on"
    , Space
    , Str "this"
    , Space
    , Str "instrument,"
    , Space
    , Str "should"
    , Space
    , Str "better"
    , Space
    , Str "be"
    , Space
    , Str "avoided."
    ]
]
